setup-hakn/action.yaml

```diff
@@ -0,0 +1,178 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Setup chainguard-dev/hakn'
+description: |
+  This action sets up chainguard-dev/hakn Knative on the current
+  kubectl context (typically KinD from setup-kind).
+
+inputs:
+  version:
+    description: |
+      The version of Knative to install, e.g. 1.5.0
+    required: true
```

Comment:
this is weird, we require users to specify the version, but provides a default.
should just change this required to false, and note that it defaults to 1.5.0

Reply:
Done